packages/app/features/top/ChatListScreen.tsx

```diff
@@ -45,8 +45,10 @@ export default function ChatListScreen(props: Props) {
 
 export function ChatListScreenView({
   previewGroupId,
+  focusedChannelId,
```

Comment:
Could we create (or use an existing) navigation context that this is part of? Trying to avoid creating more prop drilling where possible.

Reply:
yeah, I'll stick it in a context.